doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -604,3 +587,6 @@ Documentation
   * The :ref:`ug_nrf9160_gs` by replacing the Updating the DK firmware section with a new :ref:`nrf9160_gs_installing_software` section.
     This new section includes steps for using Quick Start, a new application in `nRF Connect for Desktop`_ that streamlines the getting started process with the nRF91 Series DKs.
   * Updated the :ref:`tfm_enable_share_uart` section on :ref:`ug_nrf9160`.
```

Comment:
Also, please remove the extra colon after section in the first entry in this section.
```suggestion
  * The :ref:`tfm_enable_share_uart` section on :ref:`ug_nrf9160`.
```